earthmover/__main__.py

```diff
@@ -5,6 +5,15 @@
 
 from earthmover.earthmover import Earthmover
 
+# use a dictionary here so that command strings can be accessed with a lookup.
+#   This helps enforce usage of this structure
+ALL_COMMANDS = {
```

Comment:
Tell me more about this mapping. What is an example where the keys and values would not align?

Reply:
I don't envision there being a case where they're different. The map is an imperfect structure to accomplish what I want: it's definitely clunky to read and to deal with the list as a whole. 

The main reason I did this was futureproofing: to make it clear to anyone implementing command-handling logic that all commands come from this list. If a future developer adds a new command, they'll know to add it to the global list instead of just adding another `if args.command == 'new_command'` line.

Reply:
Possible alternative?

```
RUN = "run"
COMPILE = "compile"
DEPS = "deps"
CLEAN = "clean"

ALL_COMMANDS = [RUN, COMPILE, DEPS, CLEAN]

if args.command not in ALL_COMMANDS:
    pass
```